gpt_index/tts/base.py

```diff
@@ -0,0 +1,67 @@
+"""Text to speech module."""
+
+from typing import Optional
+
+
+class barkTTS:
```

Comment:
Can we capitalize the class name? BarkTTS

Reply:
Sure.